fuzz/fuzz_targets/coordinate.rs

```diff
@@ -0,0 +1,20 @@
+#![no_main]
+use apollo_compiler::coordinate::SchemaCoordinate;
+use libfuzzer_sys::arbitrary::Arbitrary;
+use libfuzzer_sys::arbitrary::Unstructured;
+use libfuzzer_sys::fuzz_target;
+use log::debug;
+
+fuzz_target!(|data: &[u8]| {
+    let s = String::arbitrary(&mut Unstructured::new(data)).unwrap();
+
```

Comment:
libfuzzer can call `Arbitrary` itself to convert the argument to the expected type:

```suggestion
fuzz_target!(|s: &str| {
```